api/holiday_acres_api/serializers.py

```diff
@@ -50,6 +50,35 @@ def update(self, horse, data):
         # the old instance and creating a new instance
         if "feed" in data.keys():
 
+            # validate feed amount, feed type, and unit
+            for feed in data["feed"]:
+                if feed["amount"] <= 0:
+                    print("Must be a positive value")
+                    return horse
+                if feed["amount"] >= 10:
```

Comment:
I'd recommend against doing this sort of validation. In general, I'd like to avoid making the validations "too smart" to prevent people from putting in wacky data.